pkg/virt-handler/cmd-client/client.go

```diff
@@ -179,7 +179,7 @@ func (c *VirtLauncherClient) genericSendVMICmd(cmdName string,
 		},
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 180*time.Second)
```

Comment:
what's going on here? are we having issues with commands blocking?

Reply:
I think this was just for debugging. I'm sure it'll go away later on.

Reply:
yep, I just wanted to find out why we run into a timeout